generator/graphql-kotlin-federation/src/test/kotlin/com/expediagroup/graphql/generator/federation/execution/ServiceQueryResolverTest.kt

```diff
@@ -71,7 +71,7 @@ type Query @extends
 
 type Review {
   body: String!
-  content: String @deprecated(reason : "no longer supported, replace with use Review.body instead")
+  content: String
```

Comment:
why the change in schema?

Reply:
>why the change in schema?

https://github.com/apollographql/federation-jvm/blob/main/graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/printer/ServiceSDLPrinter.java#L33

the `@deprecated` directive is hidden from the `ServiceSDLPrinter`

i don't know why with previous versions the test was passing though.